client/client_grpc_partition.go

```diff
@@ -187,28 +170,14 @@ func (c *GrpcClient) LoadPartitions(ctx context.Context, collName string, partit
 				return errors.New("context deadline exceeded")
 			default:
 			}
-			partitions, err := c.ShowPartitions(ctx, collName)
+			percentage, err := c.GetLoadingProgress(ctx, collName, partitionNames)
 			if err != nil {
 				return err
 			}
-			foundLoading := false
-			loaded := 0
-			for _, partition := range partitions {
-				if _, has := ids[partition.ID]; !has {
-					continue
-				}
-				if !partition.Loaded {
-					//Not loaded
-					foundLoading = true
-					break
-				}
-				loaded++
-			}
-			if foundLoading || loaded < len(partitionNames) {
-				time.Sleep(time.Millisecond * 100)
-				continue
+			if percentage != 100 {
```

Comment:
Just a small issue: should be `== 100` here

Reply:
My bad, updated the code style and missed it here. Fixed.